KIF Tests/BackgroundTests.m

```diff
@@ -8,11 +8,11 @@
 
 #import <KIF/KIF.h>
 
-@interface BackgroundTests : KIFTestCase
+@interface ZBackgroundTests : KIFTestCase
```

Comment:
rename back to BackgroundTests

Reply:
done in https://github.com/openium/KIF/commit/4f286cf6f0264c1b93b5b1f860f00ca0bfb43104